diff --git a/node_modules/@distube/ytdl-core/lib/info.js b/node_modules/@distube/ytdl-core/lib/info.js
index 88500fd..f06fe50 100644
--- a/node_modules/@distube/ytdl-core/lib/info.js
+++ b/node_modules/@distube/ytdl-core/lib/info.js
@@ -254,7 +254,7 @@ exports.getInfo = async (id, options) => {
     if (options.playerClients.includes("ANDROID")) clientPromises.push(fetchAndroidJsonPlayer(id, options));
 
     if (clientPromises.length > 0) {
-      const responses = await Promise.allSettled(clientPromises);
+      const responses = await Promise.all(clientPromises);
       const successfulResponses = responses
         .filter(r => r.status === "fulfilled")
         .map(r => r.value)
@@ -272,8 +272,6 @@ exports.getInfo = async (id, options) => {
     }
 
     if (options.playerClients.includes("WEB")) {
-      bestPlayerResponse = info.player_response;
-
       const formats = parseFormats(info.player_response);
       if (formats && formats.length > 0) {
         formatPromises.push(sig.decipherFormats(formats, info.html5player, options));
