diff --git a/node_modules/lavalink-client/dist/index.d.ts b/node_modules/lavalink-client/dist/index.d.ts
index 1b10f80..fba498b 100644
--- a/node_modules/lavalink-client/dist/index.d.ts
+++ b/node_modules/lavalink-client/dist/index.d.ts
@@ -1066,6 +1066,10 @@ interface PlayerJson {
     nodeSessionId?: string;
     /** The stored queue */
     queue?: StoredQueue;
+    /** Custom Data for the player get/set datastorage */
+    data: {
+        [key: string]: any;
+    };
 }
 type RepeatMode = "queue" | "track" | "off";
 interface PlayerOptions {
@@ -1491,7 +1495,7 @@ type LavalinkClientSearchPlatformResolve = "bandcamp" | "bc";
 type LavalinkSearchPlatform = "ytsearch" | "ytmsearch" | "scsearch" | "bcsearch" | LavaSrcSearchPlatform | DuncteSearchPlatform | JioSaavnSearchPlatform | LavalinkClientSearchPlatform;
 type ClientCustomSearchPlatformUtils = "local" | "http" | "https" | "link" | "uri";
 type ClientSearchPlatform = ClientCustomSearchPlatformUtils | // for file/link requests
-"youtube" | "yt" | "youtube music" | "youtubemusic" | "ytm" | "musicyoutube" | "music youtube" | "soundcloud" | "sc" | "am" | "apple music" | "applemusic" | "apple" | "musicapple" | "music apple" | "sp" | "spsuggestion" | "spotify" | "spotify.com" | "spotifycom" | "dz" | "deezer" | "yandex" | "yandex music" | "yandexmusic" | "vk" | "vk music" | "vkmusic" | "tidal" | "tidal music" | "qobuz" | "flowerytts" | "flowery" | "flowery.tts" | LavalinkClientSearchPlatformResolve | LavalinkClientSearchPlatform | "js" | "jiosaavn" | "td" | "tidal" | "tdrec";
+    "youtube" | "yt" | "youtube music" | "youtubemusic" | "ytm" | "musicyoutube" | "music youtube" | "soundcloud" | "sc" | "am" | "apple music" | "applemusic" | "apple" | "musicapple" | "music apple" | "sp" | "spsuggestion" | "spotify" | "spotify.com" | "spotifycom" | "dz" | "deezer" | "yandex" | "yandex music" | "yandexmusic" | "vk" | "vk music" | "vkmusic" | "tidal" | "tidal music" | "qobuz" | "flowerytts" | "flowery" | "flowery.tts" | LavalinkClientSearchPlatformResolve | LavalinkClientSearchPlatform | "js" | "jiosaavn" | "td" | "tidal" | "tdrec";
 type SearchPlatform = LavalinkSearchPlatform | ClientSearchPlatform;
 type SourcesRegex = "YoutubeRegex" | "YoutubeMusicRegex" | "SoundCloudRegex" | "SoundCloudMobileRegex" | "DeezerTrackRegex" | "DeezerArtistRegex" | "DeezerEpisodeRegex" | "DeezerMixesRegex" | "DeezerPageLinkRegex" | "DeezerPlaylistRegex" | "DeezerAlbumRegex" | "AllDeezerRegex" | "AllDeezerRegexWithoutPageLink" | "SpotifySongRegex" | "SpotifyPlaylistRegex" | "SpotifyArtistRegex" | "SpotifyEpisodeRegex" | "SpotifyShowRegex" | "SpotifyAlbumRegex" | "AllSpotifyRegex" | "mp3Url" | "m3uUrl" | "m3u8Url" | "mp4Url" | "m4aUrl" | "wavUrl" | "aacpUrl" | "tiktok" | "mixcloud" | "musicYandex" | "radiohost" | "bandcamp" | "jiosaavn" | "appleMusic" | "tidal" | "TwitchTv" | "vimeo";
 interface PlaylistInfo {
@@ -1528,7 +1532,7 @@ interface UnresolvedSearchResult {
  * @internal
  */
 interface MiniMapConstructor {
-    new (): MiniMap<unknown, unknown>;
+    new(): MiniMap<unknown, unknown>;
     new <K, V>(entries?: ReadonlyArray<readonly [K, V]> | null): MiniMap<K, V>;
     new <K, V>(iterable: Iterable<readonly [K, V]>): MiniMap<K, V>;
     readonly prototype: MiniMap<unknown, unknown>;
diff --git a/node_modules/lavalink-client/dist/index.js b/node_modules/lavalink-client/dist/index.js
index d7b16a0..6711d07 100644
--- a/node_modules/lavalink-client/dist/index.js
+++ b/node_modules/lavalink-client/dist/index.js
@@ -4365,7 +4365,8 @@ var Player = class {
       nodeId: this.node?.id,
       nodeSessionId: this.node?.sessionId,
       ping: this.ping,
-      queue: this.queue.utils.toJSON()
+      queue: this.queue.utils.toJSON(),
+      data: this.data
     };
   }
 };
