diff --git a/node_modules/lavalink-client/dist/index.d.ts b/node_modules/lavalink-client/dist/index.d.ts
index 6557ad3..68bae5a 100644
--- a/node_modules/lavalink-client/dist/index.d.ts
+++ b/node_modules/lavalink-client/dist/index.d.ts
@@ -1074,6 +1074,10 @@ interface PlayerJson {
     nodeSessionId?: string;
     /** The stored queue */
     queue?: StoredQueue;
+    /** Custom Data for the player get/set datastorage */
+    data: {
+        [key: string]: any;
+    };
 }
 type RepeatMode = "queue" | "track" | "off";
 interface PlayerOptions {
diff --git a/node_modules/lavalink-client/dist/index.js b/node_modules/lavalink-client/dist/index.js
index 565c89d..c89b38a 100644
--- a/node_modules/lavalink-client/dist/index.js
+++ b/node_modules/lavalink-client/dist/index.js
@@ -4393,7 +4393,8 @@ var Player = class {
       nodeId: this.node?.id,
       nodeSessionId: this.node?.sessionId,
       ping: this.ping,
-      queue: this.queue.utils.toJSON()
+      queue: this.queue.utils.toJSON(),
+      data: this.data
     };
   }
 };
