diff --git a/node_modules/lavalink-client/dist/index.d.ts b/node_modules/lavalink-client/dist/index.d.ts
index 8b81c01..1110205 100644
--- a/node_modules/lavalink-client/dist/index.d.ts
+++ b/node_modules/lavalink-client/dist/index.d.ts
@@ -1328,6 +1328,10 @@ interface PlayerJson {
     nodeSessionId?: string;
     /** The stored queue */
     queue?: StoredQueue;
+    /** Custom Data for the player get/set datastorage */
+    data: {
+        [key: string]: any;
+    };
 }
 type RepeatMode = "queue" | "track" | "off";
 interface PlayerOptions {
diff --git a/node_modules/lavalink-client/dist/index.js b/node_modules/lavalink-client/dist/index.js
index 6f28f5e..65fc8a5 100644
--- a/node_modules/lavalink-client/dist/index.js
+++ b/node_modules/lavalink-client/dist/index.js
@@ -4575,7 +4575,8 @@ var Player = class {
       nodeId: this.node?.id,
       nodeSessionId: this.node?.sessionId,
       ping: this.ping,
-      queue: this.queue.utils.toJSON()
+      queue: this.queue.utils.toJSON(),
+      data: this.data
     };
   }
 };
